graphein/grn/parse_trrust.py

```diff
@@ -0,0 +1,105 @@
+# %%
+# Graphein
+# Author: Ramon Vinas, Arian Jamasb <arian@jamasb.io>
+# License: MIT
+# Project Website: https://github.com/a-r-j/graphein
+# Code Repository: https://github.com/a-r-j/graphein
+import logging
+import os
+import zipfile
+from pathlib import Path
+from typing import Any, Dict, List, Union
+
+import pandas as pd
+import wget
+
+log = logging.getLogger(__name__)
+
+
+def _download_TRRUST():
+    """
+    Downloads TRRUST
+    """
+    url = "https://www.grnpedia.org/trrust/data/trrust_rawdata.human.tsv"
+
+    # TODO: perhaps this would be better?
```

Comment:
I think we may want to specify an `out_dir` in the configs. I think if I was using this to prep data in a project, I'd like the data to be saved to a directory of my choosing in my ML project, rather than in Graphein. However, we can have a default within Graphein. Paging @ericmjl for his take.